src/transport/tcp.rs

```diff
@@ -18,8 +18,8 @@ pub struct TcpTransport {
 #[async_trait]
 impl Transport for TcpTransport {
     type Acceptor = TcpListener;
-    type Stream = TcpStream;
     type RawStream = TcpStream;
+    type Stream = TcpStream;
```

Comment:
> 为什么会有这种差异？



> Why this diff?

This modification only changes the order of the structure members, so that all Transport implementations have the same order of structure member names, maybe a bit picky